packages/@lwc/ssr-compiler/src/compile-template/transformers/component/lwc-component.ts

```diff
@@ -6,33 +6,40 @@
  */
 import { is } from 'estree-toolkit';
 import { isUndefined } from '@lwc/shared';
-import { expressionIrToEs } from '../expression';
-import { esTemplate, esTemplateWithYield } from '../../estemplate';
-import { getChildAttrsOrProps } from '../shared';
-import type { Transformer } from '../types';
+import { expressionIrToEs } from '../../expression';
+import { esTemplateWithYield } from '../../../estemplate';
+import { getChildAttrsOrProps } from '../../shared';
+import { getSlottedContent } from './slotted-content';
+import type { Transformer } from '../../types';
 import type {
     LwcComponent as IrLwcComponent,
     Expression as IrExpression,
 } from '@lwc/template-compiler';
-import type {
-    IfStatement as EsIfStatement,
-    VariableDeclaration as EsVariableDeclaration,
-} from 'estree';
-
-const bDynamicComponentConstructorDeclaration = esTemplate`
-    const Ctor = '${/*lwcIs attribute value*/ is.expression}';
-`<EsVariableDeclaration>;
+import type { Statement as EsStatement } from 'estree';
 
+// Note that `slottedContent` is the name of the variable used to store the slotted content
+// generated by getSlottedContent.
 const bYieldFromDynamicComponentConstructorGenerator = esTemplateWithYield`
+    const Ctor = '${/* lwcIs attribute value */ is.expression}';
     if (Ctor) {
         if (typeof Ctor !== 'function' || !(Ctor.prototype instanceof LightningElement)) {
             throw new Error(\`Invalid constructor: "\${String(Ctor)}" is not a LightningElement constructor.\`)
         }
         const childProps = __getReadOnlyProxy(${/* child props */ is.objectExpression});
         const childAttrs = ${/* child attrs */ is.objectExpression};
-        yield* Ctor[SYMBOL__GENERATE_MARKUP](null, childProps, childAttrs);
+        ${/* slotted content */ is.statement}
+
+        const scopeToken = hasScopedStylesheets ? stylesheetScopeToken : undefined;
+
+        yield* Ctor[SYMBOL__GENERATE_MARKUP](
+            null, 
+            childProps,
+            childAttrs,
+            slottedContent,
+            scopeToken
+        );
```

Comment:
The only thing I don't like about this is that it isn't clear where `slottedContent` comes from, I added a note to help make it clearer.